src/content/developers/tutorials/understand-the-erc-20-token-smart-contract/index.md

```diff
@@ -181,6 +181,8 @@ contract ERC20Basic is IERC20 {
     }
 }
 
+// note that the SafeMath library is not required for Solidity v0.8 and above.
+
```

Comment:
Should we just remove SafeMath entirely here?

Maybe point to it as further reading?

Reply:
@minimalsm sure - we can. I was trying to be minimal - the contracts on that page are also using Solidity v0.6.0 so they need to be updated for Solidity v0.8.0 too if we want to remove SafeMath altogether. Happy to do this if you think it is neater to have both tutorials using v0.8.0.